modules/concept-docs/pages/durability-replication-failure-considerations.adoc

```diff
@@ -21,6 +21,7 @@ include::7.0@sdk:shared:partial$durability-replication-failure-considerations.ad
 include::7.0@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=performance]
 
 include::7.0@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=txns]
+* xref:project-docs:distributed-transactions-java-release-notes.adoc[Transactions Release Notes].
```

Comment:
Is this in the right place?
Should it be in the Further Reading section?

Reply:
It was originally in the linked `tag=txns` partial, so this refactor leaves it in the same bulleted list.

Reply:
(intent is to be able to link the same partial from e.g. PHP sdk, without any Java specifics)

Reply:
My digging into the changes without building got a little tangled  :)

Reply:
Makes sense to me, unless we think this wasn't the right place for it to begin with?